TLC59116.h

```diff
@@ -46,8 +46,7 @@ class TLC59116Manager;
   - Output pins ("channels") are numbered 0..15.
   - Channels are "sinks", acting like Arduino pins that are LOW, even for PWM.
     - Connect the minus side of LEDs to the output pins.
-    - Instead of HIGH/LOW like DigitalWrite(), we use on/off/set.
-    - Channels are considered ON, OFF, , instead of HIGH/LOW (or, at some PWM value)
+    - Channels are considered ON, or OFF, instead of HIGH/LOW (or, at some PWM value)
```

Comment:
extra line seemed extraneous? Thanks for typo fix.

Reply:
(note to self: add ref to set())

Reply:
Extra line was most likely added by mistake.  Yes, extraneous.